cmd/finalize/finalize.go

```diff
@@ -27,9 +28,10 @@ type Message struct {
 func main() {
 	config := NewConfig()
 	mq := broker.New(config.Broker)
-	db, err := postgres.NewDB(config.Postgres)
-	if err != nil {
-		log.Println("err:", err)
+	dbs := postgres.NewDB(config.Postgres)
+	db := dbs.Db
+	if dbs.Error != nil {
```

Comment:
You don't need to check the error here. The NewDB function checks all errors and panics if something stupid happens.

Reply:
Not panics, but rather returns `nil` DB along with the `error`. At least, I personally prefer to avoid panicking and push the errors upwards instead.